qiskit_nature/operators/second_quantization/quadratic_hamiltonian.py

```diff
@@ -0,0 +1,306 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021, 2022.
```

Comment:
I believe that, because of the way how our copyright checker works, this is going to fail once merged to `main` because on that branch a single squashed commit dated to 2022 will be the only commit history related to this file.

@manoelmarques is there anything reasonable we can do about that?

Reply:
I did not believe we checked the validity as such of the earliest date the code was copyrighted - unless that's the only date in there since its the same as the last date it was updated, due to code getting moved around across repos/refactored etc its kinda difficult to do. The time the code was last updated, the latter (or sole) date, we did check to make sure it was current with the changes.